unit_tests/test_gce_image_filter.py

```
@@ -0,0 +1,115 @@
+# See LICENSE for more details.
+#
+# Copyright (c) 2026 ScyllaDB
+
+"""Unit tests for GCE image filter construction."""
+
+import pytest
+
+from sdcm.utils.version_utils import parse_scylla_version_tag
+
+
+def build_gce_filter(version: str = None) -> str:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)